Foundation/NSURLRequest.swift

```diff
@@ -308,15 +308,39 @@ public class NSURLRequest : NSObject, NSSecureCoding, NSCopying, NSMutableCopyin
 */
 public class NSMutableURLRequest : NSURLRequest {
     
+    private var _HTTPMethod: String? = "GET"
+    
     public required init?(coder aDecoder: NSCoder) {
         NSUnimplemented()
     }
-    /*! 
+    
+    /*!
+     @method initWithURL:
+     @abstract Initializes an NSMutableURLRequest with the given URL.
+     @discussion Default values are used for cache policy
+     (NSURLRequestUseProtocolCachePolicy) and timeout interval (60
+     seconds).
+     @param URL The URL for the request.
+     @result An initialized NSMutableURLRequest.
+     */
+    public init(URL: NSURL) {
```

Comment:
Hey sorry I think I deleted that comment too quick, didn't know you'd get to it that fast.  NSMutableURLRequest wasn't overriding NSURLRequest's private blank initializer, so the convenience initializers weren't being passed down